apps/platform-app/src/app/[locale]/(profile)/(components)/CookieBanner/CookieBanner.tsx

```diff
@@ -0,0 +1,52 @@
+'use client'
+import { Button } from '@gdh/ui-system'
+import { useEffect, useState } from 'react'
+import styles from './CookieBanner.module.scss'
+
+export const CookieBanner: React.FC = () => {
+  const [isBannerVisible, setIsBannerVisible] = useState<boolean>(false)
+
+  useEffect(() => {
+    try {
+      const hasAcceptedCookies = localStorage.getItem('cookiesAccepted')
+      if (!hasAcceptedCookies) {
+        setIsBannerVisible(true)
+      }
+    } catch (error) {
+      console.error('Could not access localStorage:', error)
+    }
+  }, [])
+
+  const acceptCookies = () => {
+    try {
+      localStorage.setItem('cookiesAccepted', 'true')
+      setIsBannerVisible(false)
+    } catch (error) {
+      console.error('Could not set cookiesAccepted in localStorage:', error)
+    }
+  }
+
+  const declineCookies = () => {
+    setIsBannerVisible(false)
+  }
+
+  if (!isBannerVisible) return null
```

Comment:
**suggestion**: Consider using a more descriptive variable name for `isBannerVisible` to clarify its purpose, such as `isCookieBannerVisible`.